Alert/Alert.js

```diff
@@ -167,15 +169,23 @@ const AlertBase = kind({
 				noTitle: (type === 'fullscreen') && !title
 			},
 			type
-		)
+		),
+		overflow: () => {
+			if (typeof window !== 'undefined') {
+				return window.innerWidth < ri.scale(1200); // If you will change this value, please change @sand-alert-overlay-content-width too.
+			}
+
+			return false;
```

Comment:
![image](https://user-images.githubusercontent.com/34676178/153353797-f498557f-25c0-47ae-b126-d2b88e762426.png)
![image](https://user-images.githubusercontent.com/34676178/153354329-63c75697-41e4-4766-afe3-bd2506b486c9.png)
![image](https://user-images.githubusercontent.com/34676178/153353916-b9e52487-fe62-4dd1-b2cc-93f87c0789c6.png)

It is still clipped on some resolution :(
Should we add the button container width?

```suggestion
		overflow: () => {
			if (typeof window !== 'undefined') {
				const buttonContainerWidth = 0 //buttons? ri.scale(1098) : 0; // @sand-button-max-width: 900px and there are some margin.
				return window.innerWidth < ri.scale(1200) + buttonContainerWidth; // If you will change this value, please change @sand-alert-overlay-content-width too.
			}

			return false;
```

Reply:
Ok, I updated to use `@sand-button-min-width` value for minimizing the side effect.